src/light-clients/ethereum.md

```diff
@@ -0,0 +1,143 @@
+# Ethereum
+
+In this chapter, we will discuss how the Ethereum light client works, as well as some advanced computer science concepts to understand why our (cryptographic) proofs are correct. We will start by analyzing the light client from afar, observing the full protocol, and then inspecting each individual step.
+
+## Protocol
+
+A light client offers a few public functions that can be called by any party. Each of these functions progresses it's inner state `S`. By this we mean, that calling one of these functions will make the light client store data, which it uses in subsequent operations.
+
+1. Initialization: can only be called once when creating the client.
+1. Updating: succively called for each block we wish to verify
+1. Freezing: called once in case of misbehaviour.
+
+`Initialization` often happens during contract deployment. Here we pass the initial state to the contract. This state either includes the genesis block, or an arbitrary block. We refer to this as the 'trusted height', which is trusted by the client to be a correct block. Often a governance proposal is used to verify that this block is correct before passing it to the light client.
+
+# Ethereum Light Client
+
+This chapter explores the architecture and operation of the Ethereum light client, along with the cryptographic foundations that ensure the validity of its proofs. We'll begin with a high-level overview of the protocol before diving into the technical details of each component.
+
+## Protocol Overview
+
+A light client exposes several public functions that can be invoked by external parties. Each function call modifies the client's internal state `S`, storing critical data that informs subsequent operations. The light client lifecycle can be divided into three distinct phases:
+
+1. **Initialization**: Occurs exactly once when the client is created
+1. **Updating**: Called repeatedly to verify each new block
+1. **Freezing**: Invoked once if malicious behavior is detected
+
+Initialization typically happens during contract deployment for smart contract-based light clients. During this phase, we provide the client with its initial trusted state, which can be either:
```

Comment:
If our goal is to explain concepts within IBC context, I would explain that this happens during a client creation phase rather than during the contract deployment. Because technically in IBC, the lc contracts are deployed, registered, then initialized with the initial state.